src/transformers/expect.test.ts

```diff
@@ -169,6 +183,10 @@ test('maps expect matchers', () => {
       expect(stuff).not.toHaveBeenCalled();
       expect(stuff).not.toHaveBeenCalled();
       expect(stuff).toHaveBeenCalledWith('foo', 'bar');
+
+      expect(mySpy.mock.calls).toEqual([['foo', 'bar']]);
```

Comment:
Worth double checking this one. I believe it should be `mySpy.mock.calls[0]`, but we miss checking that it is only called once with those arguments.

Reply:
the nested array implicitly asserts that `mySpy.mock.calls.length === 1`

Reply:
I didn't realize the array was nested. All good then. 👍🏼